src/main/java/mekanism/common/integration/MekanismHooks.java

```diff
@@ -173,6 +186,49 @@ public void loadOCDrivers()
 		} catch(Exception e) {}
 	}
 
+	@Method(modid = CYCLIC_MOD_ID)
+	public void registerCyclicCombinerOreRecipe(String ore, int quantity, Item input2, String outputName)
+	{
+		Item outputItem = ForgeRegistries.ITEMS.getValue(new ResourceLocation(CYCLIC_MOD_ID, outputName));
+		if(outputItem == Items.AIR) {
```

Comment:
that's not the default key as far as I can find (unless it has changed recently)

Reply:
Oops, you're right.